### PR TITLE
Adds syndicate fish to the traitor kit, for Aquatic warfare

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Adds the syndicate fish kit to the uplink, which includes both the Emulsijack and the Donkfish.

## Why It's Good For The Game

I had this idea of the ultimate syndicate sabotage: Dropping the toxic fish into someone's aquarium and watching it emulsify the rest of the fish into paste. We must take our battles to every level of Nanotrasen's scale, macro to micro. I WILL fuck your aquarium. Or maybe you just want to make a light hearted aquarium for your syndiebros. Both are cool with me!

## Changelog

:cl:
add: Traitors can buy the syndicate fish kit with their tc.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
The ARC want to know your location.

**Comment by newfren** [comment]
surely this can cost 1

**Comment by Fikou** [comment]
> surely this can cost 1

then people would buy 20 emulsijacks. that would be unbalanced

**Comment by tralezab** [comment]
> > surely this can cost 1
> 
> then people would buy 20 emulsijacks. that would be unbalanced

this

File: code/modules/uplink/uplink_items/badass.dm [hunk 1]
```diff
@@ -45,6 +45,15 @@
...
 	illegal_tech = FALSE
 
+/datum/uplink_item/badass/fish
+	name = "Syndicate Fish Case"
+	desc = "A fish kit containing fish lab grown in a vat. They may not all be pretty, but they're a great display of company pride. \
+			Small note of warning: The emulsijack should not be placed in an aquarium with other fish, unless your intent is to emulsify \
+			the other aquatic lifeforms into goopy fishfood. That could be a very noble goal if you drop it into a Nanotrasen aquarium."
+	progression_minimum = 5 MINUTES
+	item = /obj/item/storage/fish_case/syndicate
+	cost = 2
```
Comment (MrDoomBringer) [inline code/modules/uplink/uplink_items/badass.dm:55]:
```suggestion
	cost = 1
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items/badass.dm (hunk 1 above)
Comment (MrDoomBringer) [reply code/modules/uplink/uplink_items/badass.dm:55]: ;)

Comment (tralezab) [reply code/modules/uplink/uplink_items/badass.dm:55]: D-Double the emulsijacks!?

**Comment by ErdinyoBarboza** [comment]
ok now add an aquarium to the command lounge and make a progtot mission to drop an Emulsijack

File: code/modules/uplink/uplink_items/badass.dm (hunk 1 above)
Comment (tralezab) [reply code/modules/uplink/uplink_items/badass.dm:55]: okay fine

**Comment by Burgermancoding** [comment]
ok cool pr but where is cripwalking burgerman does not forget

**Comment by Arturlang** [comment]
Why not just make it 1tc and limit the amount of stock?

**Comment by TheBonded** [comment]
this is so sad, why do all the funny prs go stale?

**Comment by tralezab** [comment]
i have no idea why it removed stale but I assume no maintainer interest

**Pull request closed without merging**